wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -0,0 +1,146 @@
+The 4th CJB osu!mania 4K Tournament
+=========================
+
+![CJB 4K 4th](banner.png)
+
+The **The 4th CJB osu!mania 4K Tournament _(CJB 4K 4th)_**, is the forth one-on-one osu!mania 4K specified Chinese regional tournament, which hosted by CJK team. All the beatmaps used in this tournament are osu!mania 4K beatmap.
```

Comment:
Remove the comma before `is`.

`forth` -> `fourth`

Remove the comma before `which`.

`which hosted by` -> `which is hosted by the`

~`All the beatmaps used in this tournament are osu!mania 4K beatmap.`~ I think the first sentence explains that already.